vespalib/src/vespa/vespalib/trace/tracenode.h

```diff
@@ -50,7 +50,7 @@ class TraceNode {
 
     TraceNode & operator =(const TraceNode &);
     TraceNode(TraceNode &&) noexcept;
-    TraceNode & operator =(TraceNode &&) noexcept = default;
+    TraceNode & operator =(TraceNode &&) = default;
```

Comment:
I'm guessing this is due to `vespalib::string`? One of these days we really need to get around to move-enabling it... Been on my backlog for a while.